inc/namespace.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Bootstraps the Javascript Consent API.
+ *
+ * @package altis/consent
+ */
+
+namespace Altis\Consent;
+
+defined( 'WP_CONSENT_API_URL' ) or define( 'WP_CONSENT_API_URL', plugin_dir_url( __FILE__ ) );
+defined( 'WP_CONSENT_API_VERSION' ) or define( 'WP_CONSENT_API_VERSION', get_plugin_data()['Version'] ) . ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? '-' . time() : '';
+
+/**
+ * Get the plugin data from the file header.
+ */
+function get_plugin_data() {
+	return get_file_data( __FILE__, [ 'Version' => 'Version' ], false );
+}
+
+/**
+ * Enqueue and localize the javascript.
+ */
+function enqueue_api() {
+	$min     = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? '' : '.min';
+	$options = get_option( 'cookie_consent_options', [] );
+
+	wp_enqueue_script( 'altis-consent-api', WP_CONSENT_API_URL . "src/wp-consent-api$min.js", [], WP_CONSENT_API_VERSION, true );
+
+	/*
+	 * When the consenttype (optin or optout) can be set dynamically, we can tell plugins to wait in the javascript until the consenttype has been determined.
+	 */
+	$waitfor_consent_hook = apply_filters( 'wp_consent_api_waitfor_consent_hook', false );
+
+	$expiration   = $options['cookie_expiration'] ?: 30;
+	$prefix       = apply_filters( 'wp_consent_cookie_prefix', 'wp_consent' );
```

Comment:
These filters need docblocks adding. Also because they’re filtered they should ideally be validated / sanitised before output in the localize script function call.